github/messages_test.go

```diff
@@ -476,6 +476,10 @@ func TestParseWebHook(t *testing.T) {
 			payload:     &StatusEvent{},
 			messageType: "status",
 		},
+		{
```

Comment:
Please keep these sorted by moving these four lines up between "Security..." and "Star..." just like we did in `messages.go`.  Thanks.

Reply:
Like this? `{
			payload:     &SecurityAndAnalysisEvent{},
			messageType: "security_and_analysis",
		},
		{
			payload:     &StatusEvent{},
			messageType: "status",
		},
		{
			payload:     &StarEvent{},
			messageType: "star",
		},` and excuse mistakes

Reply:
No.

Like this:

```go
		{
			payload:     &SecretScanningAlertEvent{},
			messageType: "secret_scanning_alert",
		},
		{
			payload:     &SecurityAdvisoryEvent{},
			messageType: "security_advisory",
		},
		{
			payload:     &SecurityAndAnalysisEvent{},
			messageType: "security_and_analysis",
		},
		{
			payload:     &SponsorshipEvent{},
			messageType: "sponsors",
		},
		{
			payload:     &StarEvent{},
			messageType: "star",
		},
```